configuration/mysql/add_v2xhub_user.bash

```diff
@@ -2,6 +2,16 @@
 # Fail when any command fails
 #set -e
 
+# Check if V2XHUB_VERSION is passed
+if [ -n "$1" ]; then
+    V2XHUB_VERSION="$1"
+else
+    # Prompt user for V2XHUB_VERSION
+    read -p "Enter the deployed V2X-Hub version number: " V2XHUB_VERSION
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2162

Reply:
Thanks for catching that. Updated the code now.